packages/core/src/amazonqTest/chat/controller/controller.ts

```diff
@@ -860,16 +867,14 @@ export class TestController {
                 isSupportedLanguage: true,
                 credentialStartUrl: AuthUtil.instance.startUrl,
                 result: 'Succeeded',
+                httpStatusCode: '200',
             })
             telemetry.ui_click.emit({ elementId: 'unitTestGeneration_rejectDiff' })
         }
 
         await this.sessionCleanUp()
-        // TODO: revert 'Accepted' to 'Skip build and finish' once supported
-        const message = step === FollowUpTypes.RejectCode ? 'Rejected' : 'Accepted'
 
-        this.messenger.sendMessage(message, data.tabID, 'prompt')
-        this.messenger.sendMessage(`Unit test generation workflow is completed.`, data.tabID, 'answer')
+        // this.messenger.sendMessage(`Unit test generation workflow is completed.`, data.tabID, 'answer')
```

Comment:
Existing public plugin does not reach this part of code as IDE throws an error at [sessionCleanUp](https://github.com/aws/aws-toolkit-vscode/pull/6187/files#diff-13539861ea7a453e6a39c791cd44a2decdaeb64824a68f487fddbddd106f3866L1307)

Reply:
We need to check whether we need to have this text after Accept or Reject action?
As JB does show this text after Accept/Reject but VSC don't.
We can add this accordingly.